ha-scenes.yaml

```diff
@@ -547,44 +621,55 @@ variables:
+      "Glitz and glam":{"c":[[0.45,0.198],[0.5856,0.3736],[0.4007,0.2296],[0.4469,0.273],[0.5565,0.3924]],"b":0.502,"s":"Romantic"},
+      "Glowing grins":{"c":[[0.6571,0.2954],[0.5353,0.4035],[0.5939,0.379],[0.6309,0.3339],[0.3134,0.5923]],"b":0.471,"s":"Halloween"},
       "Golden pond":{"c":[[0.5063,0.4474],[0.5584,0.4083],[0.5695,0.3999],[0.482,0.4489],[0.496,0.4424]],"b":0.647,"s":"Cozy"},
+      "Golden star":{"c":[[0.6813,0.3034],[0.4812,0.4456],[0.5717,0.3975],[0.5057,0.4345],[0.6233,0.3222]],"b":0.459,"s":"Winter holidays"},
       "Hal":{"c":[[0.5953,0.3092],[0.1589,0.2701],[0.2135,0.2508],[0.4846,0.4442],[0.5641,0.3328]],"b":0.757,"s":"Futuristic"},
       "Hazy days":{"c":[[0.4228,0.4112],[0.4832,0.4251],[0.4451,0.4199],[0.4981,0.4249],[0.4625,0.4217]],"b":0.867,"s":"Pure"},
+      "Hocus pocus":{"c":[[0.1969,0.0688],[0.5938,0.261],[0.5535,0.4119],[0.1664,0.5595],[0.6915,0.3083]],"b":0.451,"s":"Halloween"},
       "Honolulu":{"c":[[0.6364,0.2963],[0.5753,0.3275],[0.5636,0.3912],[0.4759,0.4283],[0.5725,0.3601]],"b":0.392,"s":"Cozy"},
       "Horizon":{"c":[[0.1731,0.1978],[0.5182,0.4136],[0.2779,0.2188],[0.1811,0.1979],[0.5247,0.3877]],"b":0.502,"s":"Sunrise"},
       "Ibiza":{"c":[[0.5974,0.36],[0.5286,0.4068],[0.5714,0.3806],[0.4376,0.4577],[0.4787,0.4495]],"b":0.49,"s":"Party vibes"},
+      "Jolly":{"c":[[0.4925,0.4286],[0.5925,0.3248],[0.189,0.5028],[0.1559,0.1524],[0.4667,0.238]],"b":0.4,"s":"Winter holidays"},
       "Lake Placid":{"c":[[0.4216,0.3347],[0.5135,0.348],[0.4656,0.3479],[0.2444,0.253],[0.194,0.1927]],"b":0.302,"s":"Dreamy"},
       "Lake mist":{"c":[[0.248,0.2567],[0.3835,0.3261],[0.4237,0.3449],[0.4082,0.3661],[0.2263,0.2095]],"b":0.647,"s":"Peaceful"},
       "Lily":{"c":[[0.5793,0.392],[0.5099,0.2918],[0.5705,0.272],[0.4703,0.3501],[0.5407,0.4008]],"b":0.796,"s":"Lush"},
+      "Lovebirds":{"c":[[0.5528,0.3806],[0.5787,0.393],[0.5149,0.4116],[0.5014,0.4153],[0.5202,0.4369]],"b":0.396,"s":"Romantic"},
       "Magneto":{"c":[[0.4215,0.4895],[0.1593,0.1341],[0.516,0.4401],[0.1575,0.2128],[0.171,0.3389]],"b":0.878,"s":"Futuristic"},
       "Majestic morning":{"c":[[0.3979,0.413],[0.5471,0.3588],[0.4741,0.43],[0.4659,0.3561],[0.1879,0.0644]],"b":0.569,"s":"Dreamy"},
-      "Malibu pink":{"c":[[0.5573, 0.3226],[0.3678, 0.1839],[0.1561, 0.1606],[0.4548, 0.2433],[0.154, 0.0799]],"b":0.81,"s":"Special"},
       "Memento":{"c":[[0.1554,0.0996],[0.1729,0.146],[0.4964,0.4542],[0.1824,0.0807],[0.1671,0.0853]],"b":0.4,"s":"Luxurious"},
       "Meriete":{"c":[[0.4636,0.4138],[0.1685,0.055],[0.2943,0.1842],[0.1547,0.1072],[0.2092,0.1108]],"b":0.796,"s":"Lush"},
-      "Miami":{"c":[[0.3678,0.1839],[0.154,0.0799],[0.5573,0.3226],[0.1561,0.1606],[0.4548,0.2433]],"b":0.796,"s":"Party vibes"},
+      "Miami":{"c":[[0.1621,0.3061],[0.5573,0.3409],[0.5335,0.3789],[0.2007,0.2178],[0.5371,0.2735]],"b":0.753,"s":"Race Day"},
```

Comment:
It seems that Miami was part of "Party vibes", but now is in "Race Day" with different colors.

Reply:
@nilsreiter I see what has happened, there are 2 times Miami. Both in Race Day and in Party vibes.
Not sure how to solve this with current json implementation in the blueprint.

Reply:
Maybe it's time to ditch the existing format in the blueprint and just parse the other one ...